packages/info/commands.ts

```diff
@@ -0,0 +1,13 @@
+export const AVAILABLE_COMMANDS: string[] = [
```

Comment:
You use these constants inside your commands.ts, so you can keep your code DRY.

Reply:
I am not getting it.

Reply:
1. You should create an `enum` for your available commands.
2. Later you will use AVAILABLE_COMMANDS.BINARIES inside `commands.ts`

Basically it's not convenient to have the same string repeated in different places. Create ONE constant and it in those places